src/pages/index.jsx

```diff
@@ -1,3 +1,4 @@
 export { Home } from "./home/Home";
 export { Profile } from "./profile/Profile";
 export { ProductList } from "./productList/ProductList";
+export { Wishlist } from "./wishlist/Wishlist";
```

Comment:
try using relative path instead of absolute.

Reply:
I have used relative only, did'nt get you there.